client/src/main/java/org/eclipse/hono/client/HonoConnection.java

```diff
@@ -230,10 +230,19 @@ static HonoConnection newConnection(final Vertx vertx, final ClientConfigPropert
      *
      * @return The remote container id or {@code null}.
      */
-    default String getRemoteContainer() {
+    default String getRemoteContainerId() {
```

Comment:
just wondering: shouldn't this better return `null` by default? Otherwise, what purpose does it have to allow for `null` to be returned in the first place?

Reply:
I think we can remove the default implementation here - it was only added for compatibility reasons.